apis/kueue/v1beta1/workload_types.go

```diff
@@ -578,6 +578,14 @@ const (
 	// WorkloadMaximumExecutionTimeExceeded indicates that the workload exceeded its
 	// maximum execution time.
 	WorkloadMaximumExecutionTimeExceeded = "MaximumExecutionTimeExceeded"
+
+	// WorkloadWaitForPodsStart indicates that the PodsReady condition is set before
+	// the workload has started
+	WorkloadWaitForPodsStart = "WorkloadWaitForPodsStart"
```

Comment:
```suggestion
	WorkloadWaitForPodsStart = "WaitForPodsStart"
```
I think there is no need to repeat the Workload as this condition is for workloads. Same for `WorkloadWaitForPodsRecovery`